portfolio/src/main/webapp/index.html

```diff
@@ -171,11 +171,17 @@ <h2 class="mb-5">Interests</h2>
     <hr class="m-0">
     </div>
     <!-- Servlet-->
-    <section class="resume-section" id="servlet">
+    <section class="resume-section" id="comments">
       <div class="resume-section-content">
-        <h2 class="mb-5">Servlet Practice</h2>
-        <p>Click here to see a message:</p>
-        <button onclick="getMessage()">Click me!</button>
+        <h2 class="mb-5">Comments</h2>
+        <form action="/data" method="POST">
+            <p>Leave a comment:</p>
+            <input type="text" name="user-comment">
+            <br/><br/>
```

Comment:
These should be just <br>

Void elements (full list: area, base, br, col, command, embed, hr, img, input, keygen, link, meta, param, source, track, wbr), so called because they can't have content, should not be closed (go/htmlcssstyle#Document_Type).

Reply:
I fixed this so it was just <br><br>, is this what you meant? We don't have access to the style guide that you linked but I did look up the public version.